packages/components/src/mobile/keyboard-aware-flat-list/test/use-keyboard-offset.native.js

```diff
@@ -0,0 +1,224 @@
+/**
+ * External dependencies
+ */
+import { act, renderHook } from '@testing-library/react-native';
+import { Keyboard } from 'react-native';
+import RCTDeviceEventEmitter from 'react-native/Libraries/EventEmitter/RCTDeviceEventEmitter';
+
+/**
+ * Internal dependencies
+ */
+import useKeyboardOffset from '../use-keyboard-offset';
+
+jest.useFakeTimers();
+
+describe( 'useKeyboardOffset', () => {
+	beforeEach( () => {
+		Keyboard.removeAllListeners( 'keyboardDidShow' );
+		Keyboard.removeAllListeners( 'keyboardDidHide' );
+	} );
+
+	it( 'returns the initial state', () => {
+		// Arrange
+		const { result } = renderHook( () => useKeyboardOffset( true ) );
```

Comment:
TIL `renderHook` exists. It is interesting [the documentation](https://testing-library.com/docs/react-testing-library/api/#renderhook) basically says "you shouldn't use this." 🤷🏻‍♂️ 

> You should prefer render since a custom test component results in more readable and robust tests since the thing you want to test is not hidden behind an abstraction.

To be clear, I am not requesting a change.

Reply:
Interesting that they have it but then tell you to use the other one 😄 

Thanks for sharing that, I'll keep it in mind for future tests 🙇‍♂️